### DISCUSSION TITLE
Is blitz supports Nextjs Internationalized Routing?

### DISCUSSION THREAD

**Discussion by karthikcodes6** [blitz-js/blitz#1971] [Q&A] [answered]
NextJS started to support internationalized routing from v10.0.0 Is that supported in blitz as well? if yes any workable example will be helpful.

**Comment by karthikcodes6** [comment]
FYI: https://nextjs.org/docs/advanced-features/i18n-routing

**Comment by flybayer** [comment, marked as answer]
Yes! We support it same as next.js: https://blitzjs.com/docs/i18n-routing

Currently I don't know any full example.

Also fyi, in the future when you have a question, please create it as a github discussion instead of an issue. :)
